base/utils/R/convert.input.R

```diff
@@ -618,6 +618,18 @@ convert.input <- function(input.id, outfolder, formatname, mimetype, site.id, st
   }
   
   #--------------------------------------------------------------------------------------------------#
+  # Check if result has empty or missing files
+  
+  result_sizes <- result %>% purrr::map(~ .x$file) %>% purrr::map(~ .x %>% file.size) %>% as.data.frame() %>% 
+    `colnames<-`(c("file_size")) %>%
```

Comment:
use of piping requires magrittr yes?  Is that a depends for utils already?

Reply:
- [ ] Style guide says to split pipes into separate lines.
- [ ] `map(~.x$file)` I think can be rewritten as `map("file")` -- that's some nice syntactic sugar from `purrr`.
- [ ] `map(.x %>% file.size)` should just be `map(file.size)` (taking into account the pipe, it's just `map(.x, file.size)`).
- [ ] `c("file_size")` in `colnames` can just be `"file_size"`. No need to use `c` for length 1 vectors

All that said, if `result` is already a `data.frame`, you should be able to do this much more simply as:

```
result %>%
  dplyr::mutate(file_size = purrr::map_dbl(file, file.size)) # _dbl makes sure output is a numeric vector 
```

Reply:
The result object is now a list of multiple dataframes which makes applying things a bit different than just handling a dataframe. Changing the map calls to map("file") and map(file.size) wouldn't get the right elements. I probably can make this cleaner though. So right now, the map(~ .x$file) gets just the file column from each dataframe in the result list (maintains a list of dataframe structure) and then the purrr::map(~ .x %>% file.size) applies file.size function to each dataframe in that list and I put that into a data frame with as.data.frame. Now that I'm thinking about it I can flatten the list of dataframe with the file columns from results and apply filesize to that. Not sure if that will simplify things much though.